hilti/toolchain/include/ast/types/struct.h

```diff
@@ -159,14 +159,6 @@ class Struct : public TypeBase, trait::isAllocable, trait::isParameterized, trai
         return types;
     }
 
-    auto ids() const {
```

Comment:
Is this simply unused? (ok to remove then)

Reply:
Yes, this is not used. I'll move this to its own commit.